code/src/store/features/api/appApi.ts

```diff
@@ -0,0 +1,10 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+export const appApi = createApi({
+  baseQuery: fetchBaseQuery({
+    baseUrl: "http://localhost:5000/",
+    credentials: "include"
+  }),
+  reducerPath: "appApi",
+  endpoints: () => ({})
```

Comment:
Maybe create test endpoint for our convinience? For example with jsonplaceholder

Reply:
used an example url